clang/docs/analyzer/checkers.rst

```diff
@@ -1703,7 +1703,11 @@ are detected:
 * Invalid 3rd ("``whence``") argument to ``fseek``.
 
 The stream operations are by this checker usually split into two cases, a success
-and a failure case. However, in the case of write operations (like ``fwrite``,
+and a failure case.
+On the success case it also assumes that the current value of ``stdout``,
+``stderr``, or ``stdin`` can't be equal to the file pointer returned by ``fopen``.
+
+In the case of write operations (like ``fwrite``,
```

Comment:
Personally I'd re-wrap the lines; but I can accept this if you think that it's important to minimize the diff footprint.

Reply:
Same feeling. But I decided to opt for minimizing the diff. I still think that I made the right call.

Reply:
Ok, then feel free to keep the short diff.